src/settings_internal.js

```diff
@@ -254,6 +254,8 @@ var PTHREADS = false;
 
 var BULK_MEMORY = false;
 
+var SIGN_EXT = true;
```

Comment:
That seems a little obscure to me still. Is FEATURE_SIGN_EXT too long? the only downside is for things like bulk memory that also impact how things are linked, it's not just a wasm feature. but maybe that's fine.

Reply:
These are all internal settings so we are free to change them later and we can bikeshed to our hearts content.

I would go for the short prefix myself, since its an internal thing, but either way is fine with me.  Can also be a followup.

Reply:
Actually it turns out we don't need this setting, the logic can just stay in feature_matrix. I backed this out for now, can revisit if we need more settings in the upcoming PRs.